pom.xml

```diff
@@ -1418,7 +1418,7 @@
           </exclusion>
           <exclusion>
             <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-log4j12</artifactId>
+            <artifactId>slf4j-reload4j</artifactId>
```

Comment:
Could we keep `slf4j-log4j12` and also add `slf4j-reload4j` to exclusions list?

Reply:
ok ~ although Hadoop 3.3.3 no longer relies on `slf4j-log4j12 `

Reply:
[be2e8cb](https://github.com/apache/spark/pull/36721/commits/be2e8cb8f55cc40b98722349da31fef698fb3c34)  fix this

Reply:
User may build and test with Hadoop 3.3.2.

Reply:
> User may build and test with Hadoop 3.3.2 by specify the exact version of Hadoop.

Got it ~ thanks @wangyum